agent/src/beerocks/monitor/monitor_rssi.cpp

```diff
@@ -241,7 +247,7 @@ void monitor_rssi::process()
                                << " delta_val=" << int(delta_val);
                 }
             }
-            if (!conf_disable_initiative_arp) {
+            if (arp_enabled() || !conf_disable_initiative_arp) {
```

Comment:
Shouldn't this be `if (arp_enabled() && !conf_disable_initiative_arp)`?
And coming to think about it, shouldn't arp_enabled() be enough? i.e. move the conf_disable_initiative_arp check to inside the arp_enabled() API.

Reply:
Yes, you right it should be `&&`. But no, it can't be inside arp_enabled() since it is two different things.

Reply:
Not sure I understand why it is two different things, but OK.